configuration/exodus-pipeline.yaml

```diff
@@ -58,6 +58,8 @@ Conditions:
     !Equals [!Ref env, prod]
   NotProd:
     !Not [!Equals [!Ref env, prod]]
+  NotDev:
+    !Not [!Equals [!Ref env, dev]]
```

Comment:
Super minor: I wish that these Not{env} conditions followed the same ordering as the Is{env} conditions

Reply:
this condition is no longer necessary as int tests will be run against all envs